README.md

```diff
@@ -11,6 +11,8 @@ Check out Inference running on a video of a football game:
 
 https://github.com/roboflow/inference/assets/37276661/05512458-8424-4ddd-b214-57204b278528
 
+[View more example projects built with Inference](./examples/).
```

Comment:
```suggestion
## 👩‍🏫 Examples

The [`/examples` directory](./examples/) contains example code for working with and extending `inference`, including HTTP and UDP client code and an insights dashboard, along with community examples (PRs welcome)!
```

Reply:
This kind of got hidden under the video. Gave it more emphasis so people would be less likely to miss it.